File: IT/Input-Output/Python/player.py
# file player.py

class Player(object):
	def __init__(self, name):
		self.name = name
		self.score = 100

	def addScore(self, incr):
		self.__score += incr

	def getName(self):
		return self.__name

	def setName(self, name):
		self.__name = name
	
	def getScore(self):
		return self.__score

	def setScore(self, score):
		self.__score = score

	name = property(getName, setName)
	score = property(getScore, setScore)

File: IT/Input-Output/Java/Player.java
/* file Player.java */

import java.io.Serializable;

public class Player implements Serializable {
	private String name;
	private int score;

	public Player(String name) {
		this.setName(name);
		this.setScore(100);
	}

	public void addScore(int incr) {
		this.score += incr;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getName() {
		return this.name;
	}

	public int getScore() {
		return this.score;
	}

	public void setScore(int score) {
		this.score = score;
	}
}

